### PR TITLE
[EOC-596] Add favicon (Apple, Android, Microsoft support)

### PR DESCRIPTION
**Pull request by adamklepacz**
I can't test if this new favicon works on landing page on Safari, cause safari blocks all local resources. It needs to be tested by Mariusz after merged and deployed to production server.

Zrzut ekranu 2019-11-5 o 08 32 51

End_of_coffee_-_Helps_you_track_all_the_stuff_you_care_about

Zrzut ekranu 2019-11-5 o 08 23 28

End_of_Coffee_-_track_your_office_inventory_i_End_of_Coffee_-_track_your_office_inventory

### CODE REVIEW

File: client/public/browserconfig.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
```
Comment (marekrozmus) [inline client/public/browserconfig.xml]: please use two spaces for tabs

File: client/public/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "EOC",
```
Comment (marekrozmus) [inline client/public/site.webmanifest]: please use two spaces tabs

File: landing/images/favicon/browserconfig.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
```
Comment (marekrozmus) [inline landing/images/favicon/browserconfig.xml]: two spaces tabs

File: landing/images/favicon/site.webmanifest [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "EOC",
```
Comment (marekrozmus) [inline landing/images/favicon/site.webmanifest]: two spaces tabs

**Comment by adamklepacz** [comment]
ChromeEnd_of_Coffee_-_track_your_office_inventory
dsadsaEnd_of_coffee_-_Helps_you_track_all_the_stuff_you_care_about_i_End_of_coffee_-_Helps_you_track_all_the_stuff_you_care_about
firefoxEnd_of_Coffee_-_track_your_office_inventory_i_End_of_Coffee_-_track_your_office_inventory
iPhone_11_Pro_Max_—_13_2_i_End_of_Coffee_-_track_your_office_inventory
landchromeItem-0
safaEnd_of_Coffee_-_track_your_office_inventory_i_End_of_Coffee_-_track_your_office_inventory

**Comment by adamklepacz** [comment]
https://github.com/sandstreamdev/eoc/pull/450#pullrequestreview-312442662 
@alemikolo 
I don't understand?

**Pull request merged**